Panels/TabbedPanels.js

```diff
@@ -61,7 +62,8 @@ const TabbedPanelsBase = kind({
 						orientation={tabOrientation}
 						tabs={tabs}
 						tabPosition={tabPosition}
-						selected={index}
+						selectedIndex={selectedIndex}
+						index={index}
```

Comment:
Should this have both `index` and `selectedIndex`? We could probably simplify this to only need one of those two. I think `index` isn't handled anymore by TabGroup.